### PR TITLE
chore: update filter style

### PR DESCRIPTION
**Pull request by JimmFly**
image

### CODE REVIEW

File: apps/web/src/components/blocksuite/workspace-header/styles.css.ts [hunk 1]
```diff
@@ -149,7 +149,24 @@ export const pageListTitleWrapper = style({
...
   alignItems: 'center',
 });
-
+export const allPageListTitleWrapper = style({
+  fontSize: 'var(--affine-font-base)',
+  color: 'var(--affine-text-primary-color)',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  '::after': {
```
Comment (pengx17) [inline apps/web/src/components/blocksuite/workspace-header/styles.css.ts:158]: Is this valid? I mean it is not inside of `selectors`

Comment (himself65) [reply apps/web/src/components/blocksuite/workspace-header/styles.css.ts:158]: I think it invalid

**Review by Himself65 — COMMENTED** [review]
update build flag

File: apps/web/src/components/blocksuite/workspace-header/styles.css.ts (hunk 1 above)
Comment (himself65) [reply apps/web/src/components/blocksuite/workspace-header/styles.css.ts:158]: it valid

**Review by Himself65 — APPROVED** [review]

**Pull request merged**